ansible/group_vars/all

```diff
@@ -36,6 +36,11 @@ whisk:
 #
 runtimesManifest: "{{ runtimes_manifest | default(lookup('file', '{{ openwhisk_home }}/ansible/files/runtimes.json') | from_json) }}"
 
+##
+# Prewarm configuration
+prewarmKind: "{{prewarm_kind | default('nodejs:6')}}"
```

Comment:
we've increasingly moved away from ansible configs to use pure config: refer to this doc https://github.com/apache/incubator-openwhisk/blob/master/docs/dev/configuration.md

it makes it more convenient in general for kube and mesos based deployments downstream.

Reply:
@markusthoemmes should we include these as examples?